src/pudl/transform/eia861.py

```diff
@@ -850,20 +856,12 @@ def clean_nerc(df: pd.DataFrame, idx_cols: list[str]) -> pd.DataFrame:
     to 'UNK'.
 
     Args:
-        df: A DataFrame with the column 'nerc_region' to be cleaned.
+        nerc_df: A DataFrame with the column 'nerc_region' to be cleaned.
         idx_cols: A list of the primary keys and `nerc_region`.
 
     Returns:
         A DataFrame with correct and clean NERC regions.
     """
-    idx_no_nerc = idx_cols.copy()
-    if "nerc_region" in idx_no_nerc:
-        idx_no_nerc.remove("nerc_region")
-
-    # Split raw df into primary keys plus nerc region and other value cols
-    nerc_df = df[idx_cols].copy()
-    other_df = df.drop(columns="nerc_region").set_index(idx_no_nerc)
-
```

Comment:
I don't see any change in the intended behavior of the function... was this code just hanging out doing nothing useful before?